src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+interface Props {
+  user: User,
```

Comment:
Consider renaming `user` prop to `userInfo` for better readability and clarity. 🤔